### PR TITLE
Virtualizer OMAG Chassis

### PR DESCRIPTION
**Pull request by wyf-1116**
Adding Virtualizer Service Chassis to OMAG server platform.

### CODE REVIEW

File: open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/main/java/org/odpi/openmetadata/virtualdataconnector/virtualiser/Virtualizer.java [hunk 1]
```diff
@@ -2,20 +2,54 @@
...
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+import java.util.logging.Logger;
 
 @SpringBootApplication
+@ComponentScan({"org.odpi.openmetadata.*"})
+@EnableSwagger2
 @Configuration
-@ComponentScan
 public class Virtualizer {
```
Comment (bogdan-sava) [inline open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/main/java/org/odpi/openmetadata/virtualdataconnector/virtualiser/Virtualizer.java]: This class is not needed anymore

File: open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/main/java/org/odpi/openmetadata/virtualdataconnector/virtualiser/admin/VirtualizerAdminResource.java [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+
+/**
+ * VirtualizerAdminResource provides the spring annotations for the server-side
+ * implementation of the administrative interface for
+ * the Virtualizer
+ */
+
+@RestController
+@RequestMapping("/open-metadata/admin-services/users/{userId}/servers/{serverName}")
+public class VirtualizerAdminResource{
```
Comment (bogdan-sava) [inline open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/main/java/org/odpi/openmetadata/virtualdataconnector/virtualiser/admin/VirtualizerAdminResource.java]: Is this a copy of admin resource ?

Comment (innerTide) [reply open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/main/java/org/odpi/openmetadata/virtualdataconnector/virtualiser/admin/VirtualizerAdminResource.java]: Deleted

File: open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/main/java/org/odpi/openmetadata/virtualdataconnector/virtualiser/KafkaVirtualiser.java [hunk 3]
```diff
@@ -61,6 +78,53 @@ public void listenToIVOMAS() {
...
+        }
+    }
+
+    private KafkaOpenMetadataTopicConnector initializeTopicConnector(Connection topicConnection) {
+        if (topicConnection != null){
+            try {
+                ConnectorBroker connectorBroker = new ConnectorBroker();
+                KafkaOpenMetadataTopicConnector topicConnector = (KafkaOpenMetadataTopicConnector) connectorBroker.getConnector(topicConnection);
+                return topicConnector;
+            } catch (Throwable error) {
```
[the remark this answers is not in the document]
Comment (innerTide) [reply open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/main/java/org/odpi/openmetadata/virtualdataconnector/virtualiser/KafkaVirtualiser.java]: Removed

File: open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/main/java/org/odpi/openmetadata/virtualdataconnector/virtualiser/KafkaVirtualiser.java [hunk 4]
```diff
@@ -61,6 +78,53 @@ public void listenToIVOMAS() {
...
+        } catch (NullPointerException e) {
+            logger.error("Configuration is invalid!", e);
+        }
+
+        try {
+            outTopicKafkaConnector = initializeTopicConnector(virtualiserKafkaConfig.getAccessServiceOutTopic());
+            outTopicKafkaConnector.start();
+        } catch (ConnectorCheckedException e){
+            logger.error("Cannot initialize the out-topic connector", e);
+        } catch (NullPointerException e) {
```
[the remark this answers is not in the document]
Comment (innerTide) [reply open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/main/java/org/odpi/openmetadata/virtualdataconnector/virtualiser/KafkaVirtualiser.java]: Removed

File: open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/main/java/org/odpi/openmetadata/virtualdataconnector/virtualiser/KafkaVirtualiser.java [hunk 5]
```diff
@@ -61,6 +78,53 @@ public void listenToIVOMAS() {
...
 
+    @PostConstruct
+    public void initializeKafkaConnector() {
+        final String actionDescription = "initialize in and out topic Kafka connectors";
+        try {
+            inTopicKafkaConnector = initializeTopicConnector(virtualiserKafkaConfig.getAccessServiceInTopic());
+            inTopicKafkaConnector.start();
+        } catch (ConnectorCheckedException e){
+            logger.error("Cannot initialize the in-topic connector", e);
+        } catch (NullPointerException e) {
```
[the remark this answers is not in the document]
Comment (innerTide) [reply open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/main/java/org/odpi/openmetadata/virtualdataconnector/virtualiser/KafkaVirtualiser.java]: Removed

File: open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/pom.xml [hunk 6]
```diff
@@ -38,47 +66,39 @@
...
             <version>${jackson.version}</version>
         </dependency>
-
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
             <version>${jackson.version}</version>
         </dependency>
-
+        <dependency>
```
[the remark this answers is not in the document]
Comment (innerTide) [reply open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/pom.xml]: Removed

File: open-metadata-implementation/governance-servers/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/configuration/properties/VirtualizationConfig.java [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+    }
+
+    public String getVirtualizationProvider() {
+        return virtualizationProvider;
+    }
+
+    public void setVirtualizationProvider(String virtualizationProvider) {
+        this.virtualizationProvider = virtualizationProvider;
+    }
+//
```
[the remark this answers is not in the document]
Comment (innerTide) [reply open-metadata-implementation/governance-servers/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/configuration/properties/VirtualizationConfig.java]: Removed

File: open-metadata-implementation/governance-servers/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/configuration/properties/OMAGServerConfig.java [hunk 8]
```diff
@@ -95,6 +95,7 @@
     private StewardshipServicesConfig stewardshipServicesConfig = null;
     private SecuritySyncConfig        securitySyncConfig        = null;
     private List<String>              auditTrail                = null;
+    private VirtualizationConfig virtualizationConfig = null;
```
[the remark this answers is not in the document]
Comment (innerTide) [reply open-metadata-implementation/governance-servers/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/configuration/properties/OMAGServerConfig.java]: Fixed

File: open-metadata-implementation/governance-servers/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/configuration/properties/VirtualizationConfig.java [hunk 9]
```diff
@@ -0,0 +1,149 @@
...
+
+    /**
+     * Copy data from template
+     *
+     * @param template
+     */
+    public VirtualizationConfig(VirtualizationConfig template) {
+        if (template != null) {
+            this.virtualizationProvider = template.virtualizationProvider;
+//            this.virtualizationConnection = template.virtualizationConnection;
```
[the remark this answers is not in the document]
Comment (innerTide) [reply open-metadata-implementation/governance-servers/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/configuration/properties/VirtualizationConfig.java]: Removed

**Comment by planetf1** [comment]
I see there is a pom clash. The change I made previously is covered in #799 - basically I removed the jacoco definition from the virtualizer pom since the version needed to be updated, and the definitions of both version (via PluginManagement) and the useage/configuration are covered in the top level pom and needed to be consistent for the Linux Foundation Sonar build.

File: open-metadata-implementation/access-services/subject-area/subject-area-server/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/server/services/SubjectAreaTermRESTServices.java [hunk 10]
```diff
@@ -7,6 +7,7 @@
 import org.odpi.openmetadata.accessservices.subjectarea.generated.entities.GlossaryTerm.GlossaryTerm;
 import org.odpi.openmetadata.accessservices.subjectarea.generated.entities.GlossaryTerm.GlossaryTermMapper;
 import org.odpi.openmetadata.accessservices.subjectarea.generated.relationships.TermAnchor.TermAnchor;
+import org.odpi.openmetadata.accessservices.subjectarea.generated.relationships.TermISATypeOFRelationship.TermISATypeOFRelationshipMapper;
```
[the remark this answers is not in the document]
Comment (innerTide) [reply open-metadata-implementation/access-services/subject-area/subject-area-server/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/server/services/SubjectAreaTermRESTServices.java]: Reverted :)

File: open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/pom.xml [hunk 11]
```diff
@@ -38,47 +66,39 @@
...
         </dependency>
-
-        <dependency>
-            <groupId>org.testng</groupId>
-            <artifactId>testng</artifactId>
-            <version>${testng.version}</version>
-            <scope>test</scope>
-        </dependency>
-
         <dependency>
```
[the remark this answers is not in the document]
Comment (innerTide) [reply open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/pom.xml]: Removed

File: open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/pom.xml [hunk 12]
```diff
@@ -88,18 +108,23 @@
...
+            <version>${slf4j.version}</version>
+            <scope>test</scope>
         </dependency>
-
         <dependency>
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
             <version>${commons-io.version}</version>
         </dependency>
+        <dependency>
```
[the remark this answers is not in the document]
Comment (innerTide) [reply open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/pom.xml]: Removed

File: open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/pom.xml [hunk 13]
```diff
@@ -15,102 +15,106 @@
         <version>1.1-SNAPSHOT</version>
     </parent>
 
-    <properties>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
-        <derby.version>10.8.3.0</derby.version>
-    </properties>
+   <properties>
```
Comment (planetf1) [inline open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/pom.xml]: I think the jacoco setup should be removed. We have jacoco defined at the top level, and the linux foundation build uses this for the sonar scan.

I did originally remove this in #595 -- did that break anything in the virtualizer?

File: open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/pom.xml [hunk 14]
```diff
@@ -15,102 +15,106 @@
         <version>1.1-SNAPSHOT</version>
     </parent>
 
-    <properties>
-        <maven.compiler.source>1.8</maven.compiler.source>
```
Comment (planetf1) [inline open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/pom.xml]: Is there a reason we have these settings here? They should already be inherited from the top level pom

File: open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/pom.xml [hunk 15]
```diff
@@ -15,102 +15,106 @@
...
 
         <dependency>
             <groupId>ch.qos.logback</groupId>
             <artifactId>logback-classic</artifactId>
             <scope>compile</scope>
         </dependency>
         <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-autoconfigure</artifactId>
-            <version>2.1.1.RELEASE</version>
```
Comment (planetf1) [inline open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/pom.xml]: As per PR above changing spring version, we should move this into dependencyManagement in the top level pom (as we have had some spring compatability issues)

File: open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/pom.xml (hunk 13 above)
Comment (innerTide) [reply open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/pom.xml]: Hi, I've removed this part. This is the old version of virtualizer which will be deprecated.

File: open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/pom.xml (hunk 15 above)
Comment (innerTide) [reply open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/pom.xml]: I've changed it to ${spring-boot.version}

File: open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/pom.xml (hunk 14 above)
Comment (innerTide) [reply open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/pom.xml]: Removed :)

File: open-metadata-implementation/governance-servers/server-chassis/server-chassis-spring/src/main/resources/application-dev.properties [hunk 16]
```diff
@@ -0,0 +1,3 @@
+# SPDX-License-Identifier: Apache-2.0
```
Comment (bogdan-sava) [inline open-metadata-implementation/governance-servers/server-chassis/server-chassis-spring/src/main/resources/application-dev.properties]: please remove this file

File: open-metadata-resources/open-metadata-deployment/charts/vdc-open/templates/igcproxy-deployment.yaml [hunk 17]
```diff
@@ -0,0 +1,41 @@
+# <!-- SPDX-License-Identifier: Apache-2.0 -->
```
Comment (bogdan-sava) [inline open-metadata-resources/open-metadata-deployment/charts/vdc-open/templates/igcproxy-deployment.yaml]: is this Virtualizer related?

File: open-metadata-resources/open-metadata-deployment/charts/vdc-open/templates/igcproxy-deployment.yaml [hunk 18]
```diff
@@ -0,0 +1,41 @@
...
+# Copyright Contributors to the Egeria project.
```
Comment (bogdan-sava) [inline open-metadata-resources/open-metadata-deployment/charts/vdc-open/templates/igcproxy-deployment.yaml]: is this Virtualizer related?

File: open-metadata-resources/open-metadata-deployment/charts/vdc-open/templates/igcproxy-deployment.yaml (hunk 17 above)
Comment (innerTide) [reply open-metadata-resources/open-metadata-deployment/charts/vdc-open/templates/igcproxy-deployment.yaml]: Nope, this is not my file. I didn't modify it.

File: open-metadata-implementation/governance-servers/virtualization-services/readme.md [hunk 19]
```diff
@@ -0,0 +1,82 @@
...
+{{url-virtualizer}}/open-metadata/admin-services/users/{{user-name}}/servers/{{server-name}}/cohorts/{{cohort-name}}
+```
+
+- Set up virtualizer configuration
+
+**POST** following JSON object 
+````json
+{
+	"class": "VirtualizationConfig",
+	"virtualizationProvider": "GaianDB",
```
Comment (mandy-chessell) [inline open-metadata-implementation/governance-servers/virtualization-services/readme.md]: I would expect a class name for the virtualizationProvider (assuming this is the connector provider for the data virtualization platform's connector).

Need to explain what these values mean so administrator knows how to set them up.

File: open-metadata-implementation/governance-servers/virtualization-services/readme.md [hunk 20]
```diff
@@ -0,0 +1,82 @@
...
+to the following address
+```
+{{url-virtualizer}}/open-metadata/admin-services/users/{{user-name}}/servers/{{server-name}}/virtualization-service/configuration
+```
+
+- Register Virtualizer in the OMAG Application Platform
+
+**POST** to the following address
+```
+{{url-virtualizer}}/open-metadata/admin-services/users/{{user-name}}/servers/{{server-name}}/virtualization-service
```
Comment (mandy-chessell) [inline open-metadata-implementation/governance-servers/virtualization-services/readme.md]: Why do you need this call when the previous call could activate the virtualization service.

File: open-metadata-implementation/governance-servers/virtualization-services/readme.md [hunk 21]
```diff
@@ -0,0 +1,82 @@
...
+```
+
+- Start the instance of the OMAG services
```
Comment (mandy-chessell) [inline open-metadata-implementation/governance-servers/virtualization-services/readme.md]: OMAG Server rather than OMAG Services

File: open-metadata-implementation/governance-servers/virtualization-services/virtualization-services-api/src/main/java/org/odpi/openmetadata/governanceservers/virtualizationservices/event/VirtualizerTopicListener.java [hunk 22]
```diff
@@ -0,0 +1,30 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
```
Comment (mandy-chessell) [inline open-metadata-implementation/governance-servers/virtualization-services/virtualization-services-api/src/main/java/org/odpi/openmetadata/governanceservers/virtualizationservices/event/VirtualizerTopicListener.java]: Add Javadoc

File: open-metadata-implementation/governance-servers/virtualization-services/virtualization-services-server/src/main/java/org/odpi/openmetadata/governanceservers/virtualizationservices/admin/VirtualizationOperationalServices.java [hunk 23]
```diff
@@ -0,0 +1,252 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
```
Comment (mandy-chessell) [inline open-metadata-implementation/governance-servers/virtualization-services/virtualization-services-server/src/main/java/org/odpi/openmetadata/governanceservers/virtualizationservices/admin/VirtualizationOperationalServices.java]: Add Javadoc

File: open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/pom.xml [hunk 24]
```diff
@@ -103,6 +88,18 @@
             <artifactId>logback-classic</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>io.springfox</groupId>
```
Comment (mandy-chessell) [inline open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/pom.xml]: Spring fox is enabled in the server chassis - not needed here

File: open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/main/java/org/odpi/openmetadata/virtualdataconnector/virtualiser/KafkaVirtualiser.java [hunk 25]
```diff
@@ -2,13 +2,21 @@
 /* Copyright Contributors to the ODPi Egeria project. */
```
Comment (mandy-chessell) [inline open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/main/java/org/odpi/openmetadata/virtualdataconnector/virtualiser/KafkaVirtualiser.java]: Should not have any Kafka specific code in the virtualizer - should all be handled by the connector

File: open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/main/java/org/odpi/openmetadata/virtualdataconnector/virtualiser/kafka/KafkaConnector.java [hunk 26]
```diff
@@ -0,0 +1,23 @@
+/* SPDX-License-Identifier: Apache-2.0 */
```
Comment (mandy-chessell) [inline open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/main/java/org/odpi/openmetadata/virtualdataconnector/virtualiser/kafka/KafkaConnector.java]: The virtualizer should be using the open metadata kafka connector.

File: open-metadata-resources/open-metadata-deployment/charts/vdc-open/templates/igcproxy-deployment.yaml (hunk 18 above)
Comment (planetf1) [reply open-metadata-resources/open-metadata-deployment/charts/vdc-open/templates/igcproxy-deployment.yaml]: as previous - probably got pulled in due to pom change, but this is way backlevel

File: open-metadata-implementation/governance-servers/virtualization-services/readme.md (hunk 19 above)
Comment (innerTide) [reply open-metadata-implementation/governance-servers/virtualization-services/readme.md]: Hi Mandy, this is just some dummy information. I will add the items for class name and corresponding properties later.

File: open-metadata-implementation/governance-servers/virtualization-services/readme.md [hunk 27]
```diff
@@ -0,0 +1,82 @@
...
+to the following address
+```
+{{url-virtualizer}}/open-metadata/admin-services/users/{{user-name}}/servers/{{server-name}}/event-bus?connectorProvider=org.odpi.openmetadata.adapters.eventbus.topic.kafka.KafkaOpenMetadataTopicProvider&topicURLRoot={{topic-root}}
+```
+
+- Set up cohort
+
+**POST** to the following address
+```
+{{url-virtualizer}}/open-metadata/admin-services/users/{{user-name}}/servers/{{server-name}}/cohorts/{{cohort-name}}
```
[the remark this answers is not in the document]
Comment (innerTide) [reply open-metadata-implementation/governance-servers/virtualization-services/readme.md]: Hi Mandy, I tried to run the server without setting up cohort, but it returns the following error:
{
    "class": "VoidResponse",
    "relatedHTTPCode": 400,
    "exceptionClassName": "org.odpi.openmetadata.adminservices.ffdc.exception.OMAGConfigurationErrorException",
    "exceptionErrorMessage": "OMAG-ADMIN-400-009 The OMAG server virtualizer has been passed a configuration document with no open metadata repository services configuration",
    "exceptionSystemAction": "The system is unable to initialize the local server instance.",
    "exceptionUserAction": "Use the administration services to add the repository services configuration."
}

File: open-metadata-implementation/governance-servers/virtualization-services/readme.md [hunk 28]
```diff
@@ -0,0 +1,82 @@
...
+Virtualizer communicates with Information View OMAS and virtualization tool which is currently Gaian.
+
+Virtualizer has three main functions:
+1. listen to Information View OMAS Out topic(specified by property information-view-out-topic) and retrieve InformationViewEvent event (json structure);
+2. create Business Logical View(business terms are used as column names) and Technical Logical View(source table column names are used as view column names), containing only the columns with business terms assigned
+3. create Business View json file and Technical View json file, notify Information View OMAS through publishing on Information View OMAS In topic (specified by property information-view-in-topic).
+
+## OMAG server configuration
+
+Virtualizer is now running as a service for [OMAG Application Platform](../server-chassis). In this case, the service should be configured and initialized by the restful APIs provided by the platform.
```
[the remark this answers is not in the document]
Comment (innerTide) [reply open-metadata-implementation/governance-servers/virtualization-services/readme.md]: Fixed

File: open-metadata-implementation/governance-servers/virtualization-services/readme.md (hunk 27 above)
[the remark this answers is not in the document]
Comment (mandy-chessell) [reply open-metadata-implementation/governance-servers/virtualization-services/readme.md]: This suggests that the initialization is not recognizing the virtualizer configuration and needs updating to include a test for it.  It does not mean you need the cohort config

File: open-metadata-implementation/governance-servers/virtualization-services/readme.md (hunk 19 above)
Comment (mandy-chessell) [reply open-metadata-implementation/governance-servers/virtualization-services/readme.md]: It would be better to have no information than wrong information.

File: open-metadata-implementation/governance-servers/virtualization-services/virtualization-services-server/src/main/java/org/odpi/openmetadata/governanceservers/virtualizationservices/auditlog/VirtualizationAuditCode.java [hunk 29]
```diff
@@ -0,0 +1,130 @@
...
+            "The local server has completed initialization of a new instance.",
+            "No action is required.  This is part of the normal operation of the service."),
+    SERVICE_SHUTDOWN("VIRTUALIZER-0003",
+            OMRSAuditLogRecordSeverity.INFO,
+            "The Virtualizer is shutting down its instance for server {0}",
+            "The local server has requested shut down of a Virtualizer instance.",
+            "No action is required.  This is part of the normal operation of the service."),
+    IV_OUT_TOPIC_CONNECTOR_INITIALIZED("VIRTUALIZER-0004",
+            OMRSAuditLogRecordSeverity.INFO,
+            "The Virtualizer has initialized a Kafka connector for Information View Out Topic",
```
Comment (mandy-chessell) [inline open-metadata-implementation/governance-servers/virtualization-services/virtualization-services-server/src/main/java/org/odpi/openmetadata/governanceservers/virtualizationservices/auditlog/VirtualizationAuditCode.java]: Should not refer to Kafka since the event bus may not be kafka.

File: open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/main/java/org/odpi/openmetadata/virtualdataconnector/virtualiser/gaian/GaianQueryConstructor.java [hunk 30]
```diff
@@ -13,12 +13,7 @@
 import org.springframework.beans.factory.annotation.Value;
```
Comment (mandy-chessell) [inline open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/main/java/org/odpi/openmetadata/virtualdataconnector/virtualiser/gaian/GaianQueryConstructor.java]: Should not have Gaian specific code in the virtualizer

Comment (innerTide) [reply open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/main/java/org/odpi/openmetadata/virtualdataconnector/virtualiser/gaian/GaianQueryConstructor.java]: Hi Mandy, the code in the folder is the old version of virtualizer. It will be deprecated and removed from the repo when the OMAG server is done. :)

File: open-metadata-implementation/governance-servers/virtualization-services/virtualization-services-server/src/main/java/org/odpi/openmetadata/governanceservers/virtualizationservices/auditlog/VirtualizationAuditCode.java (hunk 29 above)
Comment (innerTide) [reply open-metadata-implementation/governance-servers/virtualization-services/virtualization-services-server/src/main/java/org/odpi/openmetadata/governanceservers/virtualizationservices/auditlog/VirtualizationAuditCode.java]: fixed
